src/viam/logging.py

```diff
@@ -43,31 +79,52 @@ def getLogger(name: str) -> logging.Logger:
     return logger
 
 
-def addHandlers(logger: logging.Logger):
-    logger.handlers.clear()
+def addHandlers(logger: logging.Logger, use_default_handlers=False):
+    _addHandlers([logger], use_default_handlers)
+
+
+def _addHandlers(loggers: Iterable[logging.Logger], use_default_handlers=False):
+    format = _ColorFormatter("%(asctime)s\t\t" + "%(levelname)s\t" + "%(name)s (%(filename)s:%(lineno)d)\t" + "%(message)s\t")
+
+    handlers: List[logging.Handler] = []
+
+    if _MODULE_PARENT is not None and not use_default_handlers:
+        mod_handler = _ModuleHandler(_MODULE_PARENT)
+        mod_handler.setFormatter(format)
+        mod_handler.setLevel(LOG_LEVEL)
```

Comment:
If I run the Python complex module example with your changes with `-debug` on the rdk (`go run web/cmd/server/main.go -config ../viam-configs/python_module.json -debug`), I get a _lot_ of heartbeat logs from the `RobotClient` logger. Also, when I shut down the RDK in that situation, the Python module process seems to raise some error: `grpclib.exceptions.StreamTerminatedError: Connection lost`. Any idea what's going on there?

Reply:
I don't see why the `RobotClient` logger would have _more_ heartbeat logs than it used to. They should be the same amount as before (assuming that those heartbeat logs were being printed before).
If anything, these logs would be easier to read now that they're not surrounded by all the stdout/stderr lines

Reply:
Update: these heartbeat logs were NOT being printed out before.